packages/twenty-front/src/modules/object-record/object-options-dropdown/components/ObjectOptionsDropdownMenuContent.tsx

```diff
@@ -55,6 +56,9 @@ export const ObjectOptionsDropdownMenuContent = () => {
     recordGroupFieldMetadataComponentState,
   );
 
+  const isGroupByEnabled =
+    viewType === ViewType.Kanban || currentView?.key !== 'INDEX';
```

Comment:
I would rather do something like view.viewGroups.length > 1 || currentView?.key !== 'INDEX'

Reply:
but not against viewType === ViewType.Kanban || currentView?.key !== 'INDEX'; 
just feel it's more aligned with what designs wants : if viewGroups are there let's make sure we can edit them, otherwise let's make sure we are not on an INDEX key